spec/view_objects/results_view_spec.rb

```diff
@@ -263,6 +263,11 @@
 
       it { is_expected.to eq("10") }
     end
+
+    context "when rad is not passed" do
+      let(:parameter_hash) { {} }
+      it { is_expected.to eq("20") }
+    end
```

Comment:
What about a test where the `rad` param is not in our approved list of radius?

Reply:
Cool, let me just add that!